src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java

```diff
@@ -493,9 +489,9 @@ private static HearingOptions getHearingOptions(SyaHearingOptions syaHearingOpti
                         .build();
             }
 
-            String scheduleHearing = syaHearingOptions.getScheduleHearing() ? YES : NO;
+            String scheduleHearing = Boolean.TRUE.equals(syaHearingOptions.getScheduleHearing()) ? YES : NO;
             List<ExcludeDate> excludedDates = null;
-            if (syaHearingOptions.getScheduleHearing()) {
+            if (Boolean.TRUE.equals(syaHearingOptions.getScheduleHearing())) {
```

Comment:
static import, or as above maybe revert to previous way unless Boolean object is returned

Reply:
@dattipoeHMCTS doesn't like static imports, take it up with him.. I tried to convince him.

Reply:
lol, no probs ;)